frontend/src/metabase/home/containers/HomepageApp.jsx

```diff
@@ -52,27 +48,12 @@ export default class HomepageApp extends Component {
     super(props);
     this.state = {
       greeting: Greeting.sayHello(props.user && props.user.first_name),
```

Comment:
I don't think this is used anywhere on the page.

Reply:
Removed it in https://github.com/metabase/metabase/pull/14129/commits/8f07cd0fe8738fbd9ac3d3309ae9329f31dda289. If that wasn't the right move, I can easily revert it.